distribution/rpm/src/main/packaging/init.d/elasticsearch

```diff
@@ -116,22 +115,38 @@ start() {
 
     cd $ES_HOME
     echo -n $"Starting $prog: "
-    # if not running, start it up here, usually something like "daemon $exec"
-    daemon --user $ES_USER --pidfile $pidfile $exec -p $pidfile -d -Ees.default.path.home=$ES_HOME -Ees.default.path.logs=$LOG_DIR -Ees.default.path.data=$DATA_DIR -Ees.default.path.conf=$CONF_DIR
-    retval=$?
-    echo
-    [ $retval -eq 0 ] && touch $lockfile
-    return $retval
+    # if not running, start it up here.
+
+    retval=''
+    if [ $SUSE ]; then
+        # SLES uses startproc
+        startproc -u $ES_USER $exec -p $pidfile -d -Des.default.path.home=$ES_HOME -Des.default.path.logs=$LOG_DIR -Des.default.path.data=$DATA_DIR -Des.default.path.conf=$CONF_DIR &
+        retval=$?
+        rc_status -v
+    else
+        # For non-SLES usually something like "daemon $exec"
+        daemon --user $ES_USER --pidfile $pidfile $exec -p $pidfile -d -Des.default.path.home=$ES_HOME -Des.default.path.logs=$LOG_DIR -Des.default.path.data=$DATA_DIR -Des.default.path.conf=$CONF_DIR
```

Comment:
This stomps the previous change to the init script to pass parameters using `-E`.